### ISSUE TITLE
CocoaPods After Upgrading to El Capitan And TeamCity

### ISSUE THREAD

**Issue by coridn** [CocoaPods/CocoaPods#4565]
Yesterday, I upgraded to El Capitan on our build machine.  This build machine is a Team City agent.  Prior to the upgrade, I had no issues with Cocoapods.  I had command line build steps that ran 'pod install' on the projects and all worked well.

After the upgrade to El Capitan, I started getting build failures with the following message: Cannot start build runner: Cannot run process pod install : file not found.  Strange...

I followed some of the instructions I found in another issue that addressed upgrades to El Capitan and `sudo gem uninstall cocoapods`, GEM_HOME, adding to path, etc.  I did all that but I still get a build error with the same error.

I've ran `which pod`, which returns `/usr/local/bin/pod` as expected.  TeamCity runs as the user.  
`echo $PATH` returns `/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/MacGPG2/bin:/Users/BuildAgent/Android/android-sdk-macosx/tools:/Users/BuildAgent/Android/android-sdk-macosx/platform-tools` so `pod` is in the path.  I've restarted the machine several times now to restart the TeamCity agent.  

The only way I've been able to fix it is by setting the exact path on my TeamCity step, `/usr/local/bin/pod`.
But this was never necessary before.  Does anyone understand why this is happening?

**Comment by manuyavuz** [comment]
Hi @coridn,

Could you run `echo $PATH` and `which pod` commands inside your TeamCity script step just before calling `pod install` and report what you get here?

**Comment by coridn** [comment]
Hi @manuyavuz 

Thanks for the assistance.  Printout the PATH variable in a custom script (before I tried running it as a executable with parameters which failed to print the contents of path) helped me realize that TeamCity has its own PATH environment when it runs.  I then looked at the agent's configuration/environment variables and saw that PATH does NOT contain `/usr/local/bin`.  I'll make those changes now.  Thanks!
